.github/workflows/publish-vsx-next.yml

```diff
@@ -0,0 +1,44 @@
+name: ovsx-preview-publish-vscode-built-in-extensions
+on:
+  schedule:
+    - cron: "30 0 * * *"
+  push:
+    branches:
+      - master
+env:
+  NODE_OPTIONS: --max-old-space-size=8192
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+      OVSX_TOKEN: ${{ secrets.OVSX_PAT}}
```

Comment:
The env variable should be named `OVSX_PAT`:
https://github.com/eclipse/openvsx/blob/592bf16e093cb9491cdd493d116c327e55f3304a/cli/src/publish.ts#L23

Reply:
done